configs/components/oasis3mct/oasis3mct.yaml

```diff
@@ -34,7 +35,7 @@ choose_version:
     branch: '2.8mct-awiesm-2.1'
     git-repository: https://gitlab.awi.de/paleodyn/Models/oasis3-mct.git
   '2.8':
-    branch: '2.8'
+    branch: 'v2.8'
```

Comment:
As explained in the PR message, we need to remove the obsolete branches and fix the missing ones.